assets/loadtest/k8s/Makefile

```
@@ -475,10 +490,10 @@ fetch-profiles:
 	mkdir -p $(shell pwd)/profiles/$(LOCATION)/
 	curl -o $(shell pwd)/profiles/$(LOCATION)/goroutine.profile http://127.0.0.1:3434/debug/pprof/goroutine
 	curl -o $(shell pwd)/profiles/$(LOCATION)/heap.profile http://127.0.0.1:3434/debug/pprof/heap
-	curl -o $(shell pwd)/profiles/$(LOCATION)/trace.profile http://127.0.0.1:3434/debug/pprof/trace
+	curl -o $(shell pwd)/profiles/$(LOCATION)/trace.profile http://127.0.0.1:3434/debug/pprof/trace?seconds=10
```

nit: Use `$(PWD)`, quote strings and URLs to avoid problems.

```suggestion
	curl -o "$(PWD)/profiles/$(LOCATION)/trace.profile" 'http://127.0.0.1:3434/debug/pprof/trace?seconds=10'
```